src/Symfony/Component/Security/Core/Authorization/Voter/WeightedVoter.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Security\Core\Authorization\Voter;
+
+/**
+ * WeightedVoter is an abstract implementation of a weighted voter.
+ *
+ * @author Thomas Ploch <profiploch@gmail.com>
+ */
+abstract class WeightedVoter extends Voter implements WeightedVoterInterface
```

Comment:
this class does not make sense. It does not bring any benefit compared to extending Voter and implementing the interface directly